app/javascript/application.js

```diff
@@ -1,3 +1,16 @@
+import * as Sentry from "@sentry/browser";
+
+const sentryDsn = document.querySelector("body").dataset.sentryDsn;
+if (sentryDsn) {
+  Sentry.init({
+    dsn: sentryDsn,
+    integrations: [new Sentry.BrowserTracing(), new Sentry.Replay()],
+    tracesSampleRate: 1.0,
+    replaysSessionSampleRate: 0.1,
```

Comment:
Just wondering if we need to sample any sessions without errors?

Reply:
So as I understand it the tracesSampleRate gives us that, which is mildly useful for performance tracking.

Reply:
I'm sorry, my question was super unclear - I think what I meant to ask is, why do we need to sample sessions without errors? And it was looking at `replaysSessionSampleRate: 0.1` that made me wonder that. I was thinking that config is going to generate a replay for 10% of our checks, is that something we want?

Reply:
I have tweaked this and added comments

Reply:
Ah, OK, sorry, basically errorless session monitoring is something that Sentry throws in "for free". On the back end we use it to monitor the speed of our servers and which endpoints get called the most. On the front end it adds a little more data that has the opportunity to show is if something starts going wrong. But it is maybe more than we actually need, and potentially pushes into some murky privacy water. So maybe best to remove everything except error cases.